src/libical/icalcomponent.c

```diff
@@ -936,18 +939,13 @@ void icalcomponent_foreach_recurrence(icalcomponent *comp,
     dtduration = basespan.end - basespan.start;
 
     /* Now cycle through the rrule entries */
-    for (rrule = icalcomponent_get_first_property(comp, ICAL_RRULE_PROPERTY);
+    for (;
```

Comment:
Is this line break intentional?

Reply:
The line break was left as before to reduce the size of the code change.